packages/ui/src/components/va-skip-link/VaSkipLink.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <a
+    class="va-skip-link"
+    role="link"
+    :href="target"
+    :style="position"
+  >
+    <slot />
+</a>
+</template>
+
+<script lang="ts">
+import { PropType, defineComponent } from 'vue'
+import { usePosition } from './hooks/usePosition'
+
+export default defineComponent({
+  name: 'VaSkipLink',
+  props: {
+    target: { type: String, default: '' },
+    position: {
+      type: String as PropType<'top-right' | 'top-left' | 'bottom-right' | 'bottom-left'>,
```

Comment:
Don't we have this list defined for some other component? Seems like something we can reuse.